crates/superposition/src/app_state.rs

```diff
@@ -123,7 +104,26 @@ pub async fn get(
         #[cfg(feature = "high-performance-mode")]
         redis: redis_pool,
         http_client: reqwest::Client::new(),
-        kms_client: kms_client.clone(),
-        tenant_configs,
+        encrypted_keys: join_all(get_from_env_or_default::<String>("ENCRYPTED_KEYS", String::new())
+            .split(',')
+            .map(|key| {
+                async move {
+                    let decrypted_value: String = match app_env {
+                        AppEnv::DEV | AppEnv::TEST => {
+                            get_from_env_or_default(key, "1234".into())
+                        }
+                        _ => {
+                            let kms_client = kms_client.clone().expect(
+                                "KMS client is not set. Please set the KMS client in the environment variables.",
```

Comment:
```suggestion
                                "The KMS client could not be initialized. Please check the AWS configuration for this service",
```
KMS client doesn't get set in environment variables. Can we change this message